src/index/DeltaTriples.cpp

```diff
@@ -190,8 +190,10 @@ SharedLocatedTriplesSnapshot DeltaTriples::getSnapshot() {
   // copies), hence the explicit `clone`.
   auto snapshotIndex = nextSnapshotIndex_;
   ++nextSnapshotIndex_;
+  // TODO<RobinTF> we can't clone this without violating the constraints of
+  // `LocalVocab`.
```

Comment:
This comment seems out of date. The only thing we need is a documentation in the / Shared...Snapshot class that explains why we use the `Holder`.

Reply:
Good catch, I forgot to remove this